src/main/resources/assets/js/home/xptour.ts

```diff
@@ -377,7 +377,8 @@ function enableApplicationServerEventsListener() {
         'AM',
         'applications',
     );
-    application.setPath(Path.fromString('/'));
+    const path = new PathBuilder().fromString('/').build();
```

Comment:
@poi33 Can you roll back these changes? It was a change in lib-admin-ui which broke this, but now it's fixed.